news/_posts/2024-12-22-playtest-20241222.md

```diff
@@ -0,0 +1,78 @@
+---
+kind: article
+title: "Playtest 20241222"
+author: "PunkPun"
+created_at: 2024-12-22 22:00 +0000
+disqus_id: "playtest-20241116"
```

Comment:
Is this supposed to be using the old ID?

Reply:
yes, we retain the disquss link